pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -22,6 +22,7 @@
 // Substrate macros are tripping the clippy::expect_used lint.
 #![allow(clippy::expect_used)]
 #![cfg_attr(not(feature = "std"), no_std)]
+#![feature(rustdoc_missing_doc_code_examples)]
```

Comment:
Curious why these were added.

Reply:
`make lint` was complaining about `unknown lint`. Adding the feature seemed like the least intrusive fix.